package.json

```diff
@@ -30,6 +30,13 @@
     "test:validate": "./scripts/tests-validate.sh",
     "test:browser:filter": "vitest --run --coverage false --config vitest.browser.config.mts",
     "test:e2e": "vitest --run --config vitest.node.config.mts $(scripts/tests-find.sh --e2e)",
+    "bench:node": "vitest bench --run --config vitest.node.config.mts $(scripts/tests-find.sh)",
+    "bench:node:filter": "vitest bench --run --config vitest.node.config.mts",
```

Comment:
Is it possible to run benchmarking on browser as well?

Reply:
So setting up the browser environment for the browser is a [bit more involved](https://github.com/FuelLabs/fuels-ts/actions/runs/10780020394/job/29894818040?pr=3084#step:6:108) but should be solvable. My concern though is that if the node tests take 11/12 mins on avg then adding browser setup + browser tests we could easily add another 15 mins, meaning benchmark tests could add on another 25+ mins to workflows, which is unreasonable. 

I've created [this](https://github.com/FuelLabs/fuels-ts/issues/3136) follow-up issue

Reply:
Could we not run them as separate jobs?

IMO browser performance is more important than NodeJS.